### ISSUE TITLE
Component keys cause unnecessary reflow of DOM on update

### ISSUE THREAD

**Issue by remotezygote** [georgegabe/drive-thru#4]
The use of pseudo-random keys for nodes within the app (example: https://github.com/georgegabe/drive-thru/blob/master/client/orderEditor/components/menu.jsx#L27) causes reflows and repaints of large portions of the app unnecessarily. Can you fix this to preserve non-updated DOM nodes between render cycles?

**Comment by gabxgz** [comment]
Good call. I've updated the keys to be assigned to each item just once, when they're first added to the redux store.
